hw_diag/diagnostics/bt_lte_diagnostic.py

```diff
@@ -0,0 +1,71 @@
+import os
+from hw_diag.diagnostics.diagnostic import Diagnostic
+
+
+DEVICE_TYPE_ADDRESS_MAPPINGS = [
+    {
+        'dev_type': 'BT',
+        'dev_addr': '0a12'
+    },
+    {
+        'dev_type': 'LTE',  # Quectel
+        'dev_addr': '2c7c'
+    },
+    {
+        'dev_type': 'LTE',  # Sierra Wireless MC7700
+        'dev_addr': '68a2'
+    },
+    {
+        'dev_type': 'LTE',  # Telit / Reyax
+        'dev_addr': '1bc7'
+    },
+    {
+        'dev_type': 'LTE',  # SimCom SIM7100E
+        'dev_addr': '1e0e'
+    },
+    {
+        'dev_type': 'LTE',  # Huawei ME909s-120
+        'dev_addr': '12d1'
+    },
+    {
+        'dev_type': 'LTE',  # MikroTik R11e-LTE6
+        'dev_addr': '2cd2'
+    }
+]
+
+
+class BtLteDiagnostic(Diagnostic):
+    def __init__(self, dev_type, dev_addr):
+        super(BtLteDiagnostic, self).__init__(dev_type, dev_type)
+        self.dev_addr = dev_addr
+
+    def perform_test(self, diagnostics_report):
+        resp = os.popen(
```

Comment:
I have don't like `popen` or `subprocess` and would like to see us avoid them to the greatest degree possible. I know it gets a bit more complicated to use native features like `open()` instead, but I think it will make the code a lot easier to debug.

Reply:
@vpetersson - you mean to read  /sys/bus/usb/devices/*/idVendor file as regular file using open() and then do the grepping in python code?

I can handle that once hm-config is setup on my pi. Balena cloud is timing out.

How do I test the change? test cases would be mocked? So what is the acceptance criteria for this?